internal/cli/atlas/backup/schedule/describe.go

```diff
@@ -0,0 +1,88 @@
+// Copyright 2022 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package schedule
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/mongodb/mongodb-atlas-cli/internal/cli"
+	"github.com/mongodb/mongodb-atlas-cli/internal/cli/require"
+	"github.com/mongodb/mongodb-atlas-cli/internal/config"
+	"github.com/mongodb/mongodb-atlas-cli/internal/flag"
+	"github.com/mongodb/mongodb-atlas-cli/internal/store"
+	"github.com/mongodb/mongodb-atlas-cli/internal/usage"
+	"github.com/spf13/cobra"
+)
+
+type DescribeOpts struct {
+	cli.GlobalOpts
+	cli.OutputOpts
+	clusterName string
+	store       store.ScheduleDescriber
+}
+
+func (opts *DescribeOpts) initStore(ctx context.Context) func() error {
+	return func() error {
+		var err error
+		opts.store, err = store.New(store.AuthenticatedPreset(config.Default()), store.WithContext(ctx))
+		return err
+	}
+}
+
+var scheduleDescribeTemplate = `CLUSTER NAME	AUTO EXPORT ENABLED	NEXT SNAPSHOT	SNAPSHOT HOUR	SNAPSHOT MINUTE	RESTORE WINDOW DAYS
+{{.ClusterName}}	{{.AutoExportEnabled}}	{{.NextSnapshot}}	{{.ReferenceHourOfDay}}	{{.ReferenceMinuteOfHour}}	{{.RestoreWindowDays}}
```

Comment:
judging from the ui and api response I'd say reference timings are less important than the schedule, how about listing some of the schedule data?

Reply:
Sure, it makes perfect sense, I wasn't really sure what is the desired output. I will change the template.